tabular/tests/unittests/scheduler/test_seq_scheduler.py

```diff
@@ -49,58 +49,37 @@ def test_has_enough_time_for_trial__enough_time__avg_time_not_allows_trials_by_f
 
 
 def test_LocalSequentialScheduler_no_criteria():
-    @args(lr=Real(1e-2, 1e-1, log=True))
+    search_space = {'lr': Real(1e-2, 1e-1, log=True)}
+
     def _train_fn_():
         pass
 
     with pytest.raises(AssertionError, match="Need stopping criterion: Either num_trials or time_out"):
-        LocalSequentialScheduler(train_fn=_train_fn_, reward_attr='reward_attr', resource={})
+        LocalSequentialScheduler(train_fn=_train_fn_, search_space=search_space, reward_attr='reward_attr', resource={})
 
 
 def test_search_space():
-    @ag.args(
+    search_space = dict(
         a=ag.space.Real(1e-3, 1e-2, log=True),
         b=ag.space.Real(1e-3, 1e-2),
         c=ag.space.Int(1, 10),
         d=ag.space.Categorical('a', 'b', 'c', 'd'),
         e=ag.space.Bool(),
-        f=ag.space.List(
-            ag.space.Int(1, 2),
-            ag.space.Categorical(4, 5),
-        ),
-        g=ag.space.Dict(
-            a=ag.Real(0, 10),
-            obj=ag.space.Categorical('auto', 'gluon'),
-        ),
-        h=ag.space.Categorical('test', ag.space.Categorical('auto', 'gluon')),
-        i=ag.space.Categorical('mxnet', 'pytorch'),
     )
```

Comment:
I don't think we should remove nested spaces.

Reply:
We don't support them in LocalRandomSearcher. We can add them back if we have need later.

Reply:
This is a reduction of existing functionality and I can see valid cases when nested spaces are needed: different architecture blocks with different sub-spaces in each for NAS-like HPO or even in simpler cases where we have `auto` options.
```python
{ 'epochs' ag.space.Categorical('auto', ag.space.Int(1, 100))' }
```

Reply:
I am not removing nested spaces as a class, I am removing the usage of them in this test because the local random searcher does not support nested spaces. This would have to be custom implemented and is not high priority at present.